src/cargo/util/toml.rs

```diff
@@ -814,11 +814,14 @@ fn normalize(libs: &[TomlLibTarget],
         for ex in examples.iter() {
             let path = ex.path.clone().unwrap_or_else(|| PathValue::String(default(ex)));
 
-            let profile = Profile::default_test().test(false);
-            let profile = merge(profile, &profiles.test);
+            let profile = merge(Profile::default_example(), &profiles.test);
+            let profile_release = merge(Profile::default_example_release(), &profiles.test);
```

Comment:
now when i use `bench` for release test, should i change `profiles.test` to `profiles.bench` here, @alexcrichton ? or examples should still be configured by test profile in toml